PyInstaller/utils/hooks/__init__.py

```diff
@@ -1050,6 +1050,17 @@ def collect_all(package_name, include_py_files=True):
     return datas, binaries, hiddenimports
 
 
+if is_pure_conda:
+    from . import conda as conda_support  # noqa: F401 (test import)
+elif is_conda:
+    from .conda import CONDA_META_DIR as _tmp
+    logger.warning(
+        "Assuming this isn't an Anaconda environment or an additional venv/"
+        "pipenv/... environment manager is being used on top because the "
+        "conda-meta folder %s doesn't exist." % _tmp
+    )
```

Comment:
```suggestion
    )
    del _tmp
```

Reply:
This isn't needed - `gc` will clean it up automatically (due to scopes).

Reply:
Unfortunately this will not be garbage-collected: The module globals will be kept as long as any module has a reference to this module (g.g due to some import).